tests/bluetooth/tester/src/btp_bap.c

```diff
@@ -525,6 +541,20 @@ static void stream_started(struct bt_bap_stream *stream)
 	struct audio_stream *a_stream = CONTAINER_OF(stream, struct audio_stream, stream);
 
 	LOG_DBG("Started stream %p", stream);
+
+	if (stream->dir == BT_AUDIO_DIR_SINK) {
+		a_stream->seq_num = 1;
```

Comment:
We should add a comment on why this starts at 1 instead of 0 (because the first TX is delayed by 1 interval)